docs/source/quickstart.mdx

```diff
@@ -0,0 +1,583 @@
+
+- Rover obtains the provided subgraph schemas and starts sessions for each of them.
+
+  ```sh showLineNumbers=false
+  🛫 starting a session with the 'discovery' subgraph
+  🛫 starting a session with the 'checkout' subgraph
+  🛫 starting a session with the 'users' subgraph
+  🛫 starting a session with the 'inventory' subgraph
+  🛫 starting a session with the 'orders' subgraph
+  🛫 starting a session with the 'shipping' subgraph
+  🛫 starting a session with the 'products' subgraph
+  🛫 starting a session with the 'reviews' subgraph
+  ```
+
+- Rover uses Apollo Federation to compose a supergraph schema by adding subgraph schemas one at a time.
+
+  ```sh showLineNumbers=false
+  🎶 composing supergraph with Federation v2.8.0
+  ✅ successfully composed after adding the 'products' subgraph
+  ```
+
+  - The order in which Rover adds subgraphs is undefined, so you might see composition errors like the following during intermediate steps.
+
+  ```sh showLineNumbers=false
+  error[E029]: Encountered 1 build error while trying to build a supergraph.
+  ```
+
+- When composition completes successfully, Rover starts a locally running router session and provides it with the supergraph schema.
+
+  ```sh showLineNumbers=false
+  🚀 your supergraph is running! head to http://localhost:4000 to query your supergraph
+  ```
+
+  <Note>
+
+  This message may be followed by composition errors and successful composition notifications. As long as you can access http://localhost:4000, your router is running locally.
+
+  </Note>
+
+- Rover starts watching the provided subgraph schemas for changes.
+
+  ```sh showLineNumbers=false
+  👀 watching ./subgraphs/products/schema.graphql for changes
+  ```
+
+  - Rover recomposes the supergraph schema whenever it detects one. Recomposition automatically reloads the router.
+
+Nice! You've now got a supergraph running locally.
+
+#### Query your router
+
+To ensure everything works as expected, you can query your router in the Apollo Sandbox that Rover automatically starts up.
+
+1. Open [localhost:4000](http://localhost:4000/) in a browser to access your locally running router. It should look something like this:
+
+  <img
+    src="./images/quickstart/example-sandbox.jpg"
+    className="screenshot"
+    width="700"
+    alt="The Apollo Sandbox with a prepopulated example query."
+  />
+
+2. Run the following query by copying and pasting it into the **Operation** window and then clicking the play button labeled **ExampleQuery**.
+
+```graphQL showLineNumbers=false
+query ExampleQuery {
+  listAllProducts {
+    id
+    title
+    description
+    mediaUrl
+    variants {
+      id
+      price
+    }
+  }
+}
+```
+
+3. Confirm that once you run the operation, you see the following in the **Response** panel on the right:
+
+  <ExpansionPanel title="Click to see response">
+
+  ```JSON showLineNumbers=false disableCopy=true
+  {
```

Comment:
i think there's an extra `{` and everything is indented one level too much?

Reply:
ah, right you are. Thanks for catching!